### PR TITLE
[v7r0] Add a default DIRACOS version to the integration branch -- Merge order: 1

### PR DESCRIPTION
**Pull request by chaen**
@andresailer which system should I put for the release note ?
@atsareg @zmathe can you please check ?

BEGINRELEASENOTES

*Core
NEW: Default DIRACOS version for the integration branch

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
Core?

File: releases.cfg [hunk 1]
```diff
@@ -17,6 +17,7 @@ Releases
   {
     Modules = DIRAC, Web, VMDIRAC
     Externals = v6r6p3
+    DIRACOS = master
```
Comment (fstagni) [inline releases.cfg]: Should we remove "Externals" at the same time?

Comment (zmathe) [reply releases.cfg]: I do not know if the 'integration' is used somewhere.

Comment (chaen) [reply releases.cfg]: No, I think we want to keep it as a safeguard for one version

**Comment by chaen** [comment]
Actually I am thinking it should rather be in the v6r21 branch, not the integration

**Comment by fstagni** [comment]
First produce a tag (v1r0) of DIRACOS then it can also be added to v6r21

File: releases.cfg (hunk 1 above)
Comment (atsareg) [inline releases.cfg]: Is it DIRACOS or DiracOS ?

Comment (chaen) [reply releases.cfg]: DIRACOS

Comment (atsareg) [reply releases.cfg]: def getDiracOSVersion(self, diracOSVersion=None):
    """
    It returns the DIRACOS version
    :param str diracOSVersion: the OS version
    """

    if diracOSVersion:
      return diracOSVersion
    try:
      return self.prjRelCFG[self.projectName][cliParams.release].get(
          "Releases/%s/**DiracOS**" % cliParams.release, diracOSVersion)
    except KeyError:
      pass
return diracOSVersion

Comment (chaen) [reply releases.cfg]: @zmathe you told me to use `DIRACOS`. So what is correct ?

Comment (zmathe) [reply releases.cfg]: It must be DIRACOS like: https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L746

It is a bug... going to fix it.

Comment (zmathe) [reply releases.cfg]: @chaen @atsareg Fixed in #3957

**Pull request merged**